packages/geo/src/lib/catalog/catalog-browser/catalog-browser.component.ts

```diff
@@ -81,10 +85,16 @@ export class CatalogBrowserComponent implements OnInit, OnDestroy {
       });
     }
     this.watcher = new EntityStoreWatcher(this.store, this.cdRef);
+
+    const resolution$ = this.map.viewController.resolution$;
```

Comment:
You could replace that by a simple getter 

`get resolution$(): BehaviorSubject<number> { return this.map.viewController.resolution$; }`

Reply:
done

Reply:
getting ERROR RangeError: Maximum call stack size exceeded

Find another solution.

Reply:
I don't know why you would get that error? Did you remove your subscription to resolution$? The idea is to not subscribe to the resolution simply to forward the value into another BehaviorSubject. Use the async pipe on the viewController's resolution$ (the getter enables that).